README.md

```diff
@@ -197,6 +197,19 @@ Each channel of LEDs has an `LEDStripGfx` instance associated with it. `_GFX[0]`
 
 The simplest configuration, `DEMO`, assumes you have a single meter strip of 144 LEDs and a power supply connected to your ESP32. It boots up, finds a single `RainbowFillEffect` in the `LoadEffectFactories()` function, and repeatedly calls its `Draw()` method to update the CRGB array before sending it out to the LEDs. If working correctly it should draw a scrolling rainbow palette on your LED strip.
 
+That simplest configuration, called here simply 'DEMO', is provided by a board specific buid environment. The list of such environments can be seen by running 'python3 tools/show_envs.py', which would tell the reader that, as of this writing, hardware specific variations of 'DEMO' include:
+
+- demo
+- m5demo
+- m5plusdemo
+- heltecdemo
+- heltecv2demo
+- heltecv3demo
+- lilygo-tdisplay-s3-demo
+
+These [build types](#build-pointers) may be chosen by the '-e' argument
+to pio or in a menu option inside PlatformIO/Visual Studio.

```

Comment:
I'd rephrase "PlatformIO/Visual Studio" to "the PlatformIO IDE/VS Code". The former is what the PlatformIO project mentions in their own documentation. The latter change reflects that Visual Studio Code really is something different than Visual Studio, and adopts the colloquial name most people use for it.